lib/twemproxy_exporter/counter.rb

```diff
@@ -0,0 +1,19 @@
+module TwemproxyExporter
+  class Counter
+    def initialize(registry, name, desc)
+      @counter = Prometheus::Client::Counter.new(name, desc)
+      registry.register(@counter)
+      @last = 0
+    end
+
+    def count(value, labels = {})
```

Comment:
The prometheus server should correctly handle counter resets. I think we can simplify things by dropping the counter and gauge wrapper classes and just using the prometheus ruby gem counter/gauge classes directly. A bit of more info here:
* https://prometheus.io/docs/querying/functions/#increase
* https://www.robustperception.io/how-does-a-prometheus-counter-work/